server/repository/src/migrations/v2_08_00/add_population_percentage_to_demographic.rs

```diff
@@ -0,0 +1,43 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "add_population_percentage_to_demographic"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        sql!(
+            connection,
+            r#"
+                ALTER TABLE demographic ADD COLUMN population_percentage {DOUBLE} NOT NULL DEFAULT 0;  
+            "#,
+        )?;
+
+        // Populate the new column with the existing values from demographic_indicator if they exist
```

Comment:
I found the old code was failing on remote site as the `demographic_indicator` records were missing.